tendrl/node_agent/monitoring/collectd/collectors/gluster/tendrl_gluster_brick_disk_stats.py

```diff
@@ -1,23 +1,51 @@
+import ast
+import etcd
 import psutil
 import socket
 import threading
 import time
 import traceback
 
-
+import blivet
 import collectd
-from tendrl_gluster import TendrlGlusterfsMonitoringBase
 import utils as gluster_utils
 
 
 class TendrlBrickDeviceStatsPlugin(
-    TendrlGlusterfsMonitoringBase
 ):
+    CONFIG = {}
+    b = blivet.Blivet()
+    b.reset()
+    DEVICE_TREE = b.devicetree
```

Comment:
DEVICE_TREE and blivet not used anywhere, remove them

Reply:
Aah stale unused code removing it